libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -166,6 +166,8 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_ORIENT", 53, RangeFinder, state[0].orientation, ROTATION_PITCH_270),
 
+    // @Group: _
+    // @Path: AP_RangeFinder_Wasp.cpp
```

Comment:
We can leave this for now but it's not a long-term solution because when other drivers have their own table we shouldn't mix them (there is even a possibility they might have a parameter with the same name).

Reply:
it does work with multiple backends, you just list them comma separated. Only the right docs appear in the GCS as they match against the parameters that are actually available

Reply:
having a parameter with the same name but different docs is a problem, but we can avoid that